src/test/java/io/kestra/plugin/serdes/avro/infer/InferAvroSchemaFromIonTest.java

```diff
@@ -0,0 +1,176 @@
+package io.kestra.plugin.serdes.avro.infer;
+
+import org.junit.jupiter.api.Test;
+import org.skyscreamer.jsonassert.JSONAssert;
+import org.skyscreamer.jsonassert.JSONCompareMode;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class InferAvroSchemaFromIonTest {
```

Comment:
Add a test to make sure the task is idempotent

Reply:
I could run it multiple times and check the result is the same

Reply:
would that work for you ?

Reply:
Yes, this will be OK!